department-of-veterans-affairs/abd-vro
# MCP-2433 PDF Generator Dynamic Base Path

## What was the problem?

If you ran the PDF Generator locally through `venv` instead of docker, you would have to update the hardcoded font paths from `/home/docker` to your specific path.

Associated tickets or Slack threads:

- [MCP-2433](https://amida.atlassian.net/browse/MCP-2433)

## How does this fix it?[^1]

The path is now selected automatically. If the `DOCKER` env is present then it uses `home/docker` and if not then it will look at the user's specific path instead

## How to test this PR
- Generate a PDF and make sure text is visible since the only thing using `base_path` at the moment are fonts


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
